File: HorseRacing.Web/src/interfaces/api/contracts/model/game/responses/get-available-suit/get-available-suit-response-dto.ts
import type { BaseDto } from "~/interfaces/api/contracts/base/dto/base-dto";
import type { SuitType } from "../../enums/suit-type-enum";

export interface GetAvailableSuitResponseDto extends BaseDto {
    /** Код масти */
    Suit : SuitType;
    /** Название масти */
    Name: string;
}

File: HorseRacing.Contracts/Models/Game/Responses/GetAvailableSuit/GetAvailableSuitResponseDto.cs
using HorseRacing.Contracts.Base.Dto;
using HorseRacing.Domain.GameAggregate.Enums;

namespace HorseRacing.Contracts.Models.Game.Responses.GetAvailableSuit
{
    public class GetAvailableSuitResponseDto : BaseDto
    {
        /// <summary>
        /// Код масти
        /// </summary>
        public SuitType Suit { get; set; }
        /// <summary>
        /// Название масти
        /// </summary>
        public string Name { get; set; } = string.Empty;
    }
}
